app/pages/lab-pages-editor/components/WorkflowVersion.jsx

```diff
@@ -0,0 +1,16 @@
+import { useWorkflowContext } from '../context.js';
+import LoadingIcon from '../icons/LoadingIcon.jsx'
+
+export default function WorkflowVersion({}) {
+  const { workflow, status } = useWorkflowContext();
+  const isBusy = status === 'fetching' || status === 'updating';
+  if (!workflow) return;
+
+  return (
+    <div className="workflow-version">
+      V. {workflow.version}
```

Comment:
While I moved tasks around and changed settings, the version does update 👍 Do you know off the top of your head, is this versioning only used by the classifier to check if it should re-render? Do project teams actually track version number for any other reason?

Reply:
This is now an issue for discussion on Panoptes: https://github.com/zooniverse/panoptes/issues/4340